cli/Valet/Site.php

```diff
@@ -488,7 +496,7 @@ public function secure($url, $siteConf = null, $caExpireInDays = 368)
     /**
      * If CA and root certificates are nonexistent, create them and trust the root cert.
      *
-     * @param  int  $caExpireInDays  The mount of days the self signed certificate is valid.
+     * @param  int  $caExpireInDays  The mount of days the self signed certificate authority is valid.
```

Comment:
```suggestion
     * @param  int  $caExpireInDays  The number of days the self signed certificate authority is valid.
```

Reply:
:+1: Done.

I also corrected this for the function `Site::createCertificate()`.